.github/workflows/autorelease.yaml

```diff
@@ -0,0 +1,43 @@
+name: autorelease
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 2 * * *'
```

Comment:
double-thinking, maybe it makes sense to move it to weekly. Afterall we have already nightlies, autobumps included which will run and probably choke entirely the CI. instead if we tag on weekends we get the start of week with a fresh new tag of latest

Reply:
good to me

Reply:
moved it to run at 2am each saturday